src/wp-admin/css/common.css

```
@@ -2446,25 +2452,34 @@ h1.nav-tab-wrapper, /* Back-compat for pre-4.4 */
 	line-height: 1.5;
 	cursor: pointer;
 }
+
+.accordion-section-title button.accordion-trigger:focus,
 .nav-menus-php .metabox-holder .accordion-section-title button.accordion-trigger:focus {
 	box-shadow: 0 0 0 2px #2271b1;
 	outline: 2px solid transparent;
 }
+
+.accordion-section-title span.dashicons.dashicons-arrow-down,
 .nav-menus-php .metabox-holder .accordion-section-title span.dashicons.dashicons-arrow-down {
 	position: absolute;
 	right: 10px;
 	left: auto;
 	color: #787c82;
 	border-radius: 50px;
```

This should vertically-center the arrow:
```suggestion
	border-radius: 50px;
	top: 50%;
	transform: translateY(-50%);
```

---

actually, more is needed, as this gets overridden when the accordion is open